community/rust/APKBUILD

```diff
@@ -54,7 +52,7 @@ prepare() {
 	cd "$builddir"
 
 	# Remove bundled dependencies.
-	rm -Rf src/llvm/ src/jemalloc/
+	rm -Rf src/llvm/
```

Comment:
Why did you remove it? There’s still src/jemalloc in the tarball.

Reply:
It doesn't compile without it despite of --disable-jemalloc, but I'm pretty sure jemalloc is unused

Reply:
What is the error? If it doesn’t build without it, then how you can be sure it’s not really used?

Reply:
I don't save error, but it something about missing files. I compile hello world with builded version and stripped binary about 150kb size and I don't see jemalloc string in binary